nautobot/extras/managers.py

```diff
@@ -86,6 +86,11 @@ def store_result(
             "worker": worker,
         }
 
+        job = Job.objects.get(name=task_name)
+        if job.has_sensitive_variables:
+            fields["task_args"] = ""
+            fields["task_kwargs"] = ""
```

Comment:
These are JSON fields so you probably need to just delete them from `fields`

Reply:
Yeah I'd `del` them.